web/src/main/java/com/epam/esm/config/DBConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.esm.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+
+import javax.sql.DataSource;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = "com.epam.esm")
+@EnableTransactionManagement
+public class DBConfig implements WebMvcConfigurer {
```

Comment:
Each of your modules(except domain) should include its own Configuration class. Each of these configurations has to make component scan only for this module and customize appropriate beans. In order to include configuration from one module to another one, use annotation @import.